pkg/kubelet/rkt/rkt.go

```diff
@@ -1547,8 +1715,22 @@ func (r *Runtime) GarbageCollect(gcPolicy kubecontainer.ContainerGCPolicy) error
 		if strings.HasPrefix(serviceName, kubernetesUnitPrefix) {
 			rktUUID := getRktUUIDFromServiceFileName(serviceName)
 			if _, ok := allPods[rktUUID]; !ok {
+				serviceFile := serviceFilePath(f.Name())
+				id, name, namespace, err := podDetailsFromServiceFile(serviceFile)
+				if err == nil {
+					r.cleanupPodNetwork(&api.Pod{
```

Comment:
@dcbw So this only cleans up the pod network for orphanage service. We recently added the removePod() below to remove the actual pod, so you probably want to cleanupPodNetwork there as well :)